lib/kernel/src/logger.erl

```diff
@@ -452,6 +453,27 @@ unset_module_level(Modules) ->
 unset_module_level() ->
     logger_server:unset_module_level().
 
+-spec set_application_level(Application,Level) -> ok | {error, not_loaded} when
+      Application :: atom(),
+      Level :: level() | all | none.
+set_application_level(App,Level) ->
+    case application:get_key(App, modules) of
+        {ok, Modules} ->
+            set_module_level(Modules, Level);
+        undefined ->
+            {error, not_loaded}
```

Comment:
To be similar to other error returns from other functions, like `not_found` and `already_exist`, I think this should be `{error,{not_loaded,App}}` instead. Same goes for the unset-function. And these clauses should also be tested.